runtime/src/utils/CurriedProviders.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const CurriedProviders = ({ providers, children }) =>
```

Comment:
Since each provider is actually exported by its respective `service` package with the name `Provider`, we might need to manually override the displayName if we want them to be more informative in DevTools

Reply:
Yes that is a good suggestion. Just tried setting a displayName: it works fine, but TypeScript doesn't like it. 

However, since we are already doing this
```
import { Provider as ConfigProvider } from '@dhis2/app-service-config'
import { Provider as DataProvider } from '@dhis2/app-service-data'
```
and additionally, we'd have to set a `displayName`....

Wouldn't it make more sense to just use a descriptive prefix in the exported variable name?

Or are there compelling reasons to keep the naming identical in every service?

Reply:
@HendrikThePendric I like this idea.  I went ahead and renamed all the exports from service packages (both Data and Context) to be more explicit, so they show up nicely in the React tree

<img width="952" alt="Screenshot 2019-07-09 18 06 58" src="https://user-images.githubusercontent.com/947888/60904970-5fe10800-a274-11e9-85f4-9f3e69429f38.png">

See below for discussion of top-level provider name